code/WorkInProgress/infernal_contracts.dm

```diff
@@ -491,9 +491,11 @@ obj/item/contract/satan
 		if(!..())
 			return 0
 		SPAWN_DBG(1 DECI SECOND)
+			user.unequip_all()
 			user.satanclownize()
-			boutput(user, "<span style=\"color:red; font-size:150%\"><b>Note that you are not an antagonist (unless you were already one), you simply have some of the powers of one.</b></span>")
-
+			//boutput(user, "<span style=\"color:red; font-size:150%\"><b>Note that you are not an antagonist (unless you were already one), you simply have some of the powers of one.</b></span>")
+			//this didn't actually render for the user due to the order in which these procs were called, so most people never saw this alert
+			//given the content of the OTHER big, highly visible text message, I think that moving this up would break with the current precident
```

Comment:
just del?

Reply:
wanted to leave a record in case there was administrative disagreement